v2/session/common-customizations/sessions/multiple-api-endpoints.mdx

```diff
@@ -104,15 +104,111 @@ init(
 </TabItem>
 </BackendSDKTabs>
 
-The above will set the session cookies' domain to `.example.com`, allowing them to be sent to `*.example.com`.
+The above will set the session cookies' domain to `example.com`, allowing them to be sent to `*.example.com`.
 
 :::note
 Whilst the `cookieDomain` can start with a leading `.`, the value of the `apiDomain` in `appInfo` must point to an exact API domain only. This should be the API in which you want to expose all the auth related endpoints (for example `/auth/signin`).
 
 For local development, you should not set the `cookieDomain` to an IP address based domain, or `.localhost` - browsers will reject these cookies. Instead, you should [alias `localhost` to a named domain and use that](https://superuser.com/questions/152146/how-to-alias-a-hostname-on-mac-osx). 
 :::
 
-## Step 2) Frontend config
+## Step 2) Set Older Cookie Domain in the Backend Config
+
+:::caution
+- If `olderCookieDomain` isn't set, users with older sessions will get a 500 error from the session refresh endpoint, effectively locking them out. This will continue until `olderCookieDomain` is set correctly or they clear their cookies.
```

Comment:
this caution should come below the code snippet, cause when reading this, mentioning olderCookieDomain before explaining it doesn't make much sense.

Reply:
Done